engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -24,7 +24,7 @@ sealed trait JobDescriptor[K <: CallKey] {
   def locallyQualifiedInputs: CallInputs
 }
 
-final case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
+case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
```

Comment:
"mocks are the sound of your code crying out, “please structure me differently!”"